src/util/file/file.go

```diff
@@ -35,6 +35,12 @@ func InitDataDir(dir string) (string, error) {
 		return "", err
 	}
 
+	// check if dir already exist
+	if _, err := os.Stat(dir); !os.IsNotExist(err) {
```

Comment:
this also needs to check the `stat.IsDir()`

Reply:
ok, I will add the check